### ISSUE TITLE
Auth is missing session tokken check

### ISSUE THREAD

**Issue by isuke01** [akashkaushik33/node-wp-authenticator#6]
Hey, 

I noticed Auth function is missing session tokken check for user. So it means it is completly based on cookie now.
Also if in the Wordpress Backend user click 'Log me out from other places' will not work at all since this works only on tokken.

This could be easy fixed.  Sorry I'm not cappable of fix that in your code since I'm not good enough I think to understand fully logic for now :)

Thing is: 
- You need current user meta `session_tokens` It is simply array of tokkens, where key is token
- Token part from cookie is the 3rd part (`cookieInfo[2]`)
- To generate token hash `const hashT = crypto.createHash('sha256').update(cookieInfo[2]).digest('hex');`
Then we just have to check if we find out if that token exist in our meta `session_tokens` and compare cookie time with time of token to make sure everything is correct:
`if(sesion_tokens[hashT] && Number(sesion_tokens[hashT].expiration) === Number(cookieInfo[1])) return true` 

Basically if that is true, then cooke math token for checked user (by his name - first part of the cookie) 

So it would be nice if that also could be implemented into the auth script code, since that would cover more security :) 

I hope above help you a bit with that :)

**Comment by akashkaushik33** [comment]
Hey @isuke01 , Thanks for your suggestion.  This looks interesting, I will be checking it out later.
But just a heads up, I have no immediate plans of coding this.  However, if after some time you decide to code this yourself, I can help with the understanding of the codebase. Just let me know.
Thanks.
